src/main/java/Parser.java

```diff
@@ -0,0 +1,27 @@
+public class Parser {
+
+    /**
+     * Returns the command given by the user.
+     * Accepts the string command given and create a new Comment object
+     * depending on the command type.
+     * @param command refers to the input by the user
+     * @return a new Command object
+     */
+    public Command parse(String command) {
+        String commandType = command.split(" ")[0];
+        String instruction = "";
+        String date = "";
+
+        if (commandType.equals("todo") || commandType.equals("delete") || commandType.equals("done")
+                || commandType.equals("find")) {
+            instruction = command.substring(command.indexOf(" ") + 1);
+        } else if (commandType.equals("deadline") || commandType.equals("event")) {
+            if (command.contains("/")) {
```

Comment:
This contains() method wouldn't work for commands like `event task_description /to time_date` whereby the "/at" condition was not met. Although lengthy, it would be clearer to split "event" and "deadline" into 2 different sections. 
Import java.util.regex package and use codes like `regex.matches(.\/at*)` might help. But this version is already good enough! 💃I like the clear conditions.